src/pages/groupComparison/GroupComparisonStore.ts

```diff
@@ -430,14 +430,15 @@ export default class GroupComparisonStore extends ComparisonStore {
             this.sampleKeyToSample,
             this.patients,
         ],
-        invoke: () => {
-            return Promise.resolve(
-                getCoverageInformation(
-                    this.genePanelDataForMutationProfiles.result!,
-                    this.sampleKeyToSample.result!,
-                    this.patients.result!,
-                    [this.activeMutationMapperGene!]
-                )
+        invoke: async () => {
+            if (!this.activeMutationMapperGene) {
```

Comment:
it's like, really what we need is another property on remoteData which will not fure this unless some condition is met.

Reply:
lets fix this